### ISSUE TITLE
Support IOS APP ?

### ISSUE THREAD

**Issue by Lixuya** [BananaHemic/Mumble-Unity#4]

**Comment by Bananahavoc** [comment]
I haven't personally tested on ios. To the best of my knowledge, it should work provided that there's an opus plugin for ios present, and that the import statements are correct. I encourage you to give it a shot and see if it works.

**Comment by Lixuya** [comment]
ok，I try

**Comment by Lixuya** [comment]
I used opus to compile the static library, which was wrongly reported in the Xcode project. Mistake as follows: 

Exception: Encoder error - BadArgument
  at Mumble.OpusEncoder.set_EnableForwardErrorCorrection (Boolean value) [0x00000] in <filename unknown>:0 
  at Mumble.OpusCodec.InitializeEncoderWithSampleRate (Int32 newSampleRate) [0x00000] in <filename unknown>:0 
  at MumbleTester.Start () [0x00000] in <filename unknown>:0 



have you ever met?

**Comment by Bananahavoc** [comment]
What version of Opus did you compile? Currently it's configured up to use version 1.3.

**Comment by Lixuya** [comment]
Isn't the latest version 1.1.5?                        http://www.opus-codec.org/downloads/

**Comment by Bananahavoc** [comment]
You're right, but I just haven't had the time yet to recompile for the latest version. My only thought here is that maybe the method signature changed somewhere?

**Comment by hdv0112** [comment]
I have build Opus lib and run successfully on ios. If you need, you can download it:
https://drive.google.com/open?id=1q__b2Tg1v1dgq7bmlpK_ec4ZExvxHieL

**Comment by Bananahavoc** [comment]
That's awesome news! Can you make a Pull Request?
